src/game_interpreter.cpp

```diff
@@ -1540,8 +1542,9 @@ bool Game_Interpreter::CommandGameOver(RPG::EventCommand const& /* com */) { //
 	if (Game_Message::visible) {
 		return false;
 	}
-	Game_Temp::gameover = true;
-	SetContinuation(&Game_Interpreter::DefaultContinuation);
+
+	Scene::instance->SetRequestedScene(Scene::Gameover);
+	++index;
```

Comment:
> Refactor Gameover handling 

What's this new ++index doing here?
Is that for aborting Game Over calls when another scene call overwrites in the same frame?

Reply:
That's right, otherwise we'd be stuck calling it over and over.